common-custom-user-data-maven-extension/README.md

```diff
@@ -18,6 +18,15 @@ The additional tags, links and custom values captured by this extension include:
 See [CustomBuildScanEnhancements.java](./src/main/java/com/gradle/CustomBuildScanEnhancements.java) for details on what data is
 captured and under which conditions.
 
+#### Version compatibility
+
+This table details the version compatibility of the Common Custom User Data Maven extension with the Gradle Enterprise Maven extension.
+
+| Common Custom User Data Maven extension versions | Gradle Enterprise Maven extension versions |
+| ------------------------------------------------ | ------------------------------------------ |
+| `1.5.1`                                          | `1.10.1+`                                  |
+| `1.0` - `1.5.0`                                  | `1.0.0` - `1.10`                           |
```

Comment:
This is not true.
The build cache API was introduced with version 1.6.5 according to the javadoc.
BuildScanApi was introduced with 1.0 (and it's 1.0, not 1.0.0), but some things on it came later (not sure if we want / need to go into details here, maybe just mention to look at https://docs.gradle.com/enterprise/maven-extension/api)

Reply:
Updated this accordingly.

Reply:
I fixed the table. Resolving now.